### PR TITLE
Upgrate Maestro NEXT libs

### PR DESCRIPTION
**Pull request by diegopereirawildlife**
Upgrate Maestro NEXT libs

### CODE REVIEW

File: internal/adapters/storage/postgres/scheduler/scheduler_storage_pg.go [hunk 1]
```diff
@@ -397,6 +397,7 @@ func (s schedulerStorage) RunWithTransaction(ctx context.Context, transactionFun
 	transactionID := ports.TransactionID(uuid.New().String())
 	s.transactionsMap[transactionID] = transaction
 
+	//nolint use any(nil)
```
Comment (reinaldooli) [inline internal/adapters/storage/postgres/scheduler/scheduler_storage_pg.go:400]: Instead of disable the lint, remove the `any` casting in line 402

Comment (diegopereirawildlife) [reply internal/adapters/storage/postgres/scheduler/scheduler_storage_pg.go:400]: the value that `err` receive is of type `any`, so it forces the cast

File: internal/adapters/events/forwarder_client.go [hunk 2]
```diff
@@ -167,6 +167,7 @@ func (f *ForwarderClient) createGRPCConnection(address string) (*grpc.ClientConn
 	tracer := opentracing.GlobalTracer()
 	conn, err := grpc.Dial(
 		address,
+		//nolint keep use deprecated method
```
[the remark this answers is not in the document]
Comment (diegopereirawildlife) [reply internal/adapters/events/forwarder_client.go]: Done

File: internal/adapters/storage/postgres/scheduler/scheduler_storage_pg.go (hunk 1 above)
Comment (diegopereirawildlife) [reply internal/adapters/storage/postgres/scheduler/scheduler_storage_pg.go]: I inputted the specific rule for this too

**Review by tobiasbu — APPROVED** [review]

File: internal/adapters/storage/postgres/scheduler/scheduler_storage_pg.go (hunk 1 above)
Comment (diegopereirawildlife) [reply internal/adapters/storage/postgres/scheduler/scheduler_storage_pg.go]: I removed because its just warm in my local lint

**Review by reinaldooli — APPROVED** [review]

**Pull request merged**